File: abc/abc137/b.py
# https://atcoder.jp/contests/abc137/submissions/37705183

k, x = map(int, input().split())
for i in range(x - k + 1, x + k):
  print(i, end = "")
  if i == x + k - 1:
    print()
  else:
    print(end = " ")

File: abc/abc137/b.cpp
// https://atcoder.jp/contests/abc137/submissions/27282938

#include <bits/stdc++.h>
using namespace std;
int main() {
  int k, x; cin >> k >> x;
  for(int i = x-k+1; i < x; i++) cout << i << endl;
  cout << x << endl;
  for(int i = x+1; i < x+k; i++) cout << i << endl;
}
